### ISSUE TITLE
Add an optional "List" view for landing pages

### ISSUE THREAD

**Issue by dwhathaway** [MicrosoftDocs/feedback#233]
# Add an optional "List" view for landing pages

## Issue Details

The card-based layouts on the docs landing pages are hard to parse for some users.  When there's 2 or 3 cards, it's not an issue, but when you start to see dozens of cards on a screen, it's sometimes hard to actually see the card you want to click on.  

Examples:

https://docs.microsoft.com/en-us/visualstudio/
https://docs.microsoft.com/en-us/xamarin/#pivot=platforms&panel=XamarinForms
https://docs.microsoft.com/en-us/azure/#pivot=products

Add an alternate "List" view option would make this much easier.  We don't have to remove the card view entirely, but adding an optional "switch" button that allows users to choose the format of lists would make the site much more usable.  This is already in other products like Visual Studio App Center, and HockeyApp, and one of the first things I select when logging into those tools for the first time.

**Comment by Powerhelmsman** [comment]
@dwhathaway Thanks for your suggestion. @ShevaDas Do we have any plan on the card UI design?

**Comment by ShevaDas** [comment]
Hi @dwhathaway! We don't currently have any plans to update the layout, but I've passed this request along to our design team. We're in the process of doing a site-wide UI/UX review and will make sure to keep this in mind.

**Comment by dend** [comment]
Closing this as `wont-fix` for now.
